examples/src/main/scala/com/hortonworks/spark/registry/examples/SchemaExample.scala

```diff
@@ -0,0 +1,94 @@
+package com.hortonworks.spark.registry.examples
+
+import java.util.UUID
+
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.functions.{from_json, struct, to_json}
+import org.apache.spark.sql.streaming.{OutputMode, Trigger}
+import org.apache.spark.sql.types._
+
+/**
+ * This example illustrates the usage of Spark StructType to manually define the schema
+ * and process messages from kafka. [[SchemaRegistryJsonExample]]
+ * and [[SchemaRegistryAvroExample]], illustrates how this can be simplified by
+ * integrating with Schema registry.
+ *
+ * This example depends on the resources available at
+ * https://github.com/hortonworks/registry/tree/master/examples/schema-registry
+ *
+ * To run the example:
+ * 1. Run the Spark application
+ *    SchemaExample <bootstrap-servers> <input-topic> <output-topic> <checkpoint-location>
+ * 2. Ingest the sample data (json) into input topic
+ *    $ cat examples/schema-registry/avro/data/truck_events_json | kafka-console-producer.sh
+ *    --broker-list host:port --topic topic1
+ * 3. Monitor the output topic
+ *    $ kafka-console-consumer.sh --bootstrap-server host:port --new-consumer --topic topic1-out
+ *
+ */
+object SchemaExample {
```

Comment:
`SchemaExample` -> `SchemaJsonExample`? We may add `SchemaAvroExample` for Spark 2.4 later.